docs/appendices/release-notes/5.9.6.rst

```diff
@@ -47,6 +47,9 @@ See the :ref:`version_5.9.0` release notes for a full list of changes in the
 Fixes
 =====
 
+- Fixed an ``EXPLAIN ANALYZE INSERT ...`` statement to throw an exception as it
+  is not yet supported.
```

Comment:
Please remove `yet` as this sounds it will be supported soon while it is unclear if we'll ever support it.
```suggestion
  is not supported.
```